python/py-elasticsearch/Portfile

```diff
@@ -14,11 +14,11 @@ long_description    ${description}
 
 homepage            https://github.com/elastic/elasticsearch-py
 
-checksums           rmd160  120a7ccdd6960bc22e74e3b3620a4ea019b40608 \
-                    sha256  bd6c3b8c751114f42c0265646a202da0f00f6337a9e334b4abc60ef03d1aad7c \
-                    size    303549
+checksums           rmd160  3f197083f5c342bd41aeab0e990f35f59c4320cc \
+                    sha256  d3367fc013e04fc7aad349a6de9fad1ee04fb6d627b0e7896aa505c12fde5e04 \
+                    size    313760
 
-python.versions     37 38 39 310
+python.versions     37 38 39 310 311
 
 if {${name} ne ${subport}} {
     depends_build-append \
```

Comment:
upstream lists `install_requires = ["elastic-transport>=8,<9"]` - I don't see that here, but it appears that the build succeeds. Can you please check and clarify?

Reply:
Thanks, done.